spec/features/users/user_index_spec.rb

```diff
@@ -3,21 +3,49 @@
 require "rails_helper"
 
 feature "user index", js: true do
+  include_context "search"
+  include_context "batch delete"
   let(:admin) { create(:admin) }
   let(:mission) { get_mission }
 
   before do
     login(admin)
   end
 
-  describe "batch delete" do
-    let!(:users) { create_list(:user, 3, mission: mission) }
+  describe "bulk destroy not paginated" do
+    let!(:coordinators) { create_list(:user, 5, mission: mission) }
+    let!(:enumerators) { create_list(:user, 5, mission: mission, role_name: :enumerator) }
 
-    scenario do
-      visit("/en/m/#{mission.compact_name}/users")
-      all("input.batch_op").each { |b| b.set(true) }
-      accept_confirm { click_on("Delete Multiple Users") }
-      expect(page).to have_content("3 users deleted successfully")
+    context "unfiltered" do
+      it_behaves_like "select all on page", link: "Delete Multiple Users", klass: "users", num: 11
```

Comment:
I'd like to find a way to reword these shared_examples names so that they really spell out what is being tested. It's kind of hard to read right now...

Reply:
n/a